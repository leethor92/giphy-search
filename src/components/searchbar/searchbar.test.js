import React from 'react';
import { shallow } from 'enzyme';
import Searchbar from './index';
describe('Searchbar', () => {
    it('should render correctly in "debug" mode', () => {
        const component = shallow(<Searchbar debug />);

        expect(component).toMatchSnapshot();
    });
});