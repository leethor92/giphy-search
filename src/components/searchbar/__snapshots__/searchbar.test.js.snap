// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Searchbar should render correctly in "debug" mode 1`] = `
<div
  className="container-fluid"
>
  <div
    className="row bg-primary"
  >
    <div
      className="col-md-12"
    >
      <h4
        className="text-center"
      >
        <input
          className="ui input search"
          placeholder="Search for GIFS!!!"
          type="text"
        />
        <button
          type="submit"
          value="Click to start your search"
        >
          Search
        </button>
      </h4>
    </div>
  </div>
</div>
`;
